src/Tearing/TearingScenarioEngine.inl

```diff
@@ -116,7 +116,7 @@ void TearingScenarioEngine<DataTypes>::computeEndPoints(
     Real norm_dir = dir.norm();
        
     Pb = Pa + alpha/norm_dir * dir;
-    Pc = Pa - alpha /norm_dir * dir;
+    Pc = Pa;// -alpha / norm_dir * dir;
```

Comment:
```suggestion

```
This should be kept because we want fracture in both direction from the fracture point (Pa).
And in case we are starting from a border, Pc will be out of the mesh.